### PR TITLE
Add coordination practices

### PR DESCRIPTION
**Pull request by benhylau**
This addresses https://github.com/hyphacoop/organizing/issues/132

Based on current practices and recent discussions:
- https://github.com/hyphacoop/organizing/blob/master/2018-december-retreat/2018-12-08-retreat-day-02.md#governance-and-future-plans
- https://hyphacoop.github.io/handbook/onboarding.html#our-virtual-office
- https://loomio.hypha.coop/d/p0A2fXia/when-to-use-which-communication-tool
- https://github.com/hyphacoop/organizing/blob/master/2019-10-30-all-hands-meeting.md#discussions

Related next steps:
- [Improve task workflow](https://github.com/hyphacoop/organizing/issues/140)
- [Time tracking practices](https://loomio.hypha.coop/d/iE1VYzgZ/tool-talk-time-tracking/35)

@hyphacoop/member-workers

### CODE REVIEW

File: coordination.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+| Primal Glow PG       | [label:proj:primal-glow][label:proj:primal-glow]         |                                                                              |
+
+Note that while most tasks show up as Issues from `hyphacoop/organizing` and `hyphacoop/organizing-private`, there will also be Issues from other repositories, and some items that require your attention will show up as Pull Requests. The above search filters will catch everything. It is recommended for members to bookmark the search filters that apply to them in their browser and check them everyday.
+
+Members are encouraged to use emojis liberally to indicate they have, for example: 👀 read the comment, 👍 agree or 👎 disagree with the comment, etc. This applies generally to all our tools.
+
+## Making decisions
+
+Once tasks are assigned to groups, the group is entrusted to make decisions on the task and make judgment on when it is necessary to consult the wider membership or request specific members for input. Larger group decisions are made either **at an All-hands Call and captured in meeting notes** or using **voting tools in Loomio**.
```
Comment (benhylau) [inline coordination.md]: Discuss at All-hands

File: coordination.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+- [hyphacoop/organizing](https://github.com/hyphacoop/organizing/) for regular tasks
+- [hyphacoop/organizing-private](https://github.com/hyphacoop/organizing-private/) for tasks that may contain sensitive information and are viewable by members only
+
+A task is assigned to a _Working Group (WG)_ or _Project Group (PG)_ using [Labels](https://github.com/hyphacoop/organizing/labels/), then it becomes the responsibility of the group. A task may further be assigned to a specific member. Labels are also used to indicate priorities of tasks.
+
+The following table lists all the groups and search filters that help members keep track of tasks they are responsible for.
+
+- **Assignments** mean you or your group is _responsible for the task_. The rate tasks are completed and their relative priorities are decisions by the group.
+- **Mentions** mean you or your group is _requested for input_. The response time expectation is **2 working days**, which means each member should check their Mentions at least that frequently, for their own GitHub handle and every [GitHub Team](https://github.com/orgs/hyphacoop/teams) they are part of.
```
Comment (benhylau) [inline coordination.md]: Discuss response time at All-hands

**Comment by benhylau** [comment]
For example, because I mentioned @hyphacoop/member-workers you can click this search filter and this issue will show up. You are a member of this team, so this issue is relevant to you.

Screen Shot 2019-11-12 at 3 57 07 AM

For members with limited capacity to participate, checking these filters 3 times a week is the only responsibility in addition to participating in All-hands and other committed tasks.

File: coordination.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
+# Coordination
+
+Members of Hypha adopt the following practices for mostly asynchoronous coordination using a couple digital platforms.
+
+## Primary coordination
+
+[GitHub](https://github.com/hyphacoop/) is our primary tool for coordinating asynchronously. We use [GitHub Projects](https://link.hypha.coop/tasks) to track tasks that primarily come from two repositories:
+
+- [hyphacoop/organizing](https://github.com/hyphacoop/organizing/) for regular tasks
```
Comment (patcon) [inline coordination.md]:
```suggestion
- [`hyphacoop/organizing`](https://github.com/hyphacoop/organizing/): Default tasks
```

File: coordination.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+- [hyphacoop/organizing-private](https://github.com/hyphacoop/organizing-private/) for tasks that may contain sensitive information and are viewable by members only
```
Comment (patcon) [inline coordination.md]:
```suggestion
- [`hyphacoop/organizing-private`](https://github.com/hyphacoop/organizing-private/): Confidential tasks
```

File: coordination.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+
+We use [Matrix](https://chat.tomesh.net/#/group/+hyphacoop:tomesh.net) as real-time chat for day-to-day task discussions and pinging members for urgent requests.
+
+All members are encouraged to install a Matrix client on their phone, and configure notifications for `+hyphacoop:tomesh.net` rooms to **Mentions only**. In messages, we refer to members by initials and only use handles when trying to trigger a notification to bring someone to a conversation (i.e., an urgent ping). There is otherwise no expectation that members will check the chat regularly, or catch up on backscrolls, therefore it is important to be intentional in bringing summaries of decisions made back into Issues in GitHub.
+
+## Shared calendar
+
+We have two shared Google Calendars:
+
+- [hyphacoop](https://link.hypha.coop/calendar) for _internal meetings_ and _public events of interest_
```
Comment (benhylau) [inline coordination.md]:
```suggestion
- [`hyphacoop`](https://link.hypha.coop/calendar): _Internal meetings_ and _public events of interest_
```

File: coordination.md [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+- [hyphacoop: availability](https://link.hypha.coop/availability) for member _office hours_ and _travels (as all-day events)_
```
Comment (benhylau) [inline coordination.md]:
```suggestion
- [`hyphacoop: availability`](https://link.hypha.coop/availability): _Member office hours_ and _travels (as all-day events)_
```

File: coordination.md [hunk 7]
```diff
@@ -0,0 +1,72 @@
+# Coordination
```
Comment (patcon) [inline coordination.md]:
```suggestion
# Coordination 🚧
```

My impression is that this is again writing toward a direction that many of us hope we move in, but not things that have emerged as validated practice yet. Would love to indicate that again via this emoji :)

File: SUMMARY.md [hunk 8]
```diff
@@ -5,6 +5,7 @@
   - [Values](values.md)
 * [Our Virtual Office](onboarding.md)
   - [Onboarding](onboarding.md)
+  - [Coordination](coordination.md)
```
Comment (patcon) [inline SUMMARY.md]:
```suggestion
  - [Coordination 🚧](coordination.md)
```

See https://github.com/hyphacoop/handbook/pull/29/files#r348088339

File: coordination.md [hunk 9]
```diff
@@ -0,0 +1,72 @@
...
+Members of Hypha adopt the following practices for mostly asynchoronous coordination using a couple digital platforms.
+
+## Primary coordination
+
+[GitHub](https://github.com/hyphacoop/) is our primary tool for coordinating asynchronously. We use [GitHub Projects](https://link.hypha.coop/tasks) to track tasks that primarily come from two repositories:
+
+- [`hyphacoop/organizing`](https://github.com/hyphacoop/organizing/): Default tasks
+- [`hyphacoop/organizing-private`](https://github.com/hyphacoop/organizing-private/): Confidential tasks
+
+A task is assigned to a _Working Group (WG)_ or _Project Group (PG)_ using [Labels](https://github.com/hyphacoop/organizing/labels/), then it becomes the responsibility of the group. A task may further be assigned to a specific member. Labels are also used to indicate priorities of tasks.
```
Comment (patcon) [inline coordination.md]:
```suggestion
A task is assigned to a _Working Group (WG)_ or _Project Group (PG)_ using [Labels](https://github.com/hyphacoop/organizing/labels/),
then it becomes the responsibility of the group.
A task may further be assigned to a specific member.
Labels are also used to indicate priorities of tasks.
```

File: SUMMARY.md (hunk 8 above)
Comment (benhylau) [reply SUMMARY.md]: I don't think we should put the emoji in the menu, it just makes more possibility to get out of sync.

Comment (patcon) [reply SUMMARY.md]: If it's a sync concern (which admittedly feels quite trivial), let's just put it only in menu where it is most useful for navigating.

**Comment by patcon** [comment]
@hyphacoop/infra-ops-wg discussed this and is in favour of the general direction. Let's merge this, try putting it in action, and then tweak it as we figure out what works best for us.

**Pull request merged**